### PR TITLE
RHDEVDOCS-4550 - Logging Z Stream 5.4.10 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: [RHDEVDOCS-4550](https://issues.redhat.com//browse/RHDEVDOCS-4550)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by IshwarKanse** [comment]
LGTM

File: modules/cluster-logging-rn-5.4.10.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+link:https://access.redhat.com/security/cve/CVE-2022-2058[CVE-2022-2058]
+link:https://access.redhat.com/security/cve/CVE-2022-2519[CVE-2022-2519]
+link:https://access.redhat.com/security/cve/CVE-2022-2520[CVE-2022-2520]
+link:https://access.redhat.com/security/cve/CVE-2022-2521[CVE-2022-2521]
+link:https://access.redhat.com/security/cve/CVE-2022-2867[CVE-2022-2867]
+link:https://access.redhat.com/security/cve/CVE-2022-2868[CVE-2022-2868]
+link:https://access.redhat.com/security/cve/CVE-2022-2869[CVE-2022-2869]
+link:https://access.redhat.com/security/cve/CVE-2022-2953[CVE-2022-2953]
+link:https://access.redhat.com/security/cve/CVE-2022-2964[CVE-2022-2964]
+link:https://access.redhat.com/security/cve/CVE-2022-4139[CVE-2022-4139]
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-rn-5.4.10.adoc:25]: @libander I don't see below security CVE's included in 5.4.10 errata link:https://access.redhat.com/security/cve/CVE-2022-2964[CVE-2022-2964]
link:https://access.redhat.com/security/cve/CVE-2022-4139[CVE-2022-4139]

**Review by rolfedh — APPROVED** [review]
LGTM pending Kabir's comment.

File: modules/cluster-logging-rn-5.4.10.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-rn-5.4.10.adoc:25]: They both are. That list of CVEs was copied directly from the the errata as always.

File: modules/cluster-logging-rn-5.4.10.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+[id="cluster-logging-release-notes-5-4-10_{context}"]
+= Logging 5.4.10
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0385[OpenShift Logging Bug Fix Release 5.4.10].
+
+[id="openshift-logging-5-4-10-bug-fixes"]
+== Bug fixes
+None.
+
+[id="openshift-logging-5-4-10-CVEs"]
+== CVEs
```
Comment (bburt-rh) [inline modules/cluster-logging-rn-5.4.10.adoc:12]: The list of links needs to be formatted as a bulleted list, as it is in the other sections of the release notes.

**Pull request merged**